### PR TITLE
chore: Remove required.txt

### PR DESCRIPTION
**Pull request by nschonni**
Remove references from setup and other requirements.txt files

### CODE REVIEW

**Comment by nschonni** [comment]
Linux failures seem to be an unrelated `apt` failure, but I can't restart the job. CodeCov also seems incorrect

**Comment by nijel** [comment]
I've restarted the jobs now (it should address the coverage issue as well as coverage from the Linux jobs was missing).

I've kept it in https://github.com/translate/translate/pull/3999 to keep it simple and I also wanted to open up discussion on the deps overall, see https://github.com/translate/translate/issues/4004 for that.

**Comment by nschonni** [comment]
No worries, if you end up going that way, feel free to close this. Just noticed the dead file on that other PR

**Comment by nijel** [comment]
I'm not sure what is best approach in this case, you're welcome to provide feedback in https://github.com/translate/translate/issues/4004

File: setup.py [hunk 1]
```diff
@@ -510,7 +489,6 @@ def dosetup(name, version, packages, datafiles, scripts, ext_modules=[]):
         scripts=scripts,
         ext_modules=ext_modules,
         cmdclass=cmdclass,
-        install_requires=parse_requirements('requirements/required.txt'),
```
Comment (unho) [inline setup.py:513]: This probably should be kept but instead point to `recommended.txt` like https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies @nijel thoughts on this?

Comment (unho) [reply setup.py:513]: I see that this is kinda proposed in #4004

**Comment by nijel** [comment]
Here is PR for making lxml required: https://github.com/translate/translate/pull/4035 I think it's reasonable as a lot of the code uses it, but this is something what should get broader consensus. I don't see any other dependency which should be promoted, so in case that one is not accepted, let's merge this one.

**Comment by nijel** [comment]
Closing this one as lxml will be required, see https://github.com/translate/translate/issues/4048#issuecomment-642892426 and #4035.

**Comment by nschonni** [comment]
K, not a problem

**Pull request closed without merging**